packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/remote-table.service.ts

```diff
@@ -338,6 +339,15 @@ export class RemoteTableService {
     remoteServer: RemoteServerEntity<RemoteServerType>,
     distantTableColumns: DistantTableColumn[],
   ) {
+    const referencedTable: ReferencedTable = remoteServer.schema
```

Comment:
I'd rather check on the type of the FDW here instead of relying on the existence of schema since "object" seems to be specific to Stripe FDW